torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -396,10 +383,10 @@ struct GraphFuser {
   }
 
   void run() {
-    size_t i = 0;
     for(auto p : graph->inputs()) {
-      topological_index[p] = i++;
+      topological_index[p->node()] = 0;
```

Comment:
This looks semantically different, so I just wanted to make sure this was intended.

Reply:
It is, I switched topological_index to be associated with the node rather than the value, since nodes have a topological order, and values are just defined when their node() is defined.